gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=1.41.6
+version=1.41.7
```

Comment:
Let's do a minor bump here. We are dropping support for 2.7 completely and change a lot of transient dependencies.